tools/replay/replay.cc

```diff
@@ -16,8 +16,14 @@ Replay::Replay(QString route, QStringList allow, QStringList block, SubMaster *s
   auto event_struct = capnp::Schema::from<cereal::Event>().asStruct();
   sockets_.resize(event_struct.getUnionFields().size());
   for (const auto &it : services) {
+    uint16_t which = event_struct.getFieldByName(it.name).getProto().getDiscriminantValue();
+    if ((which == cereal::Event::Which::UI_DEBUG || which == cereal::Event::Which::USER_FLAG) &&
+        !(flags & REPLAY_FLAG_ALL_SERVICES) && 
```

Comment:
maybe make a set somewhere with the blacklisted ones and then it can be included in the help dialog

Reply:
Added `base_blacklist` which is passed to Replay. Then the help message for `--all` looks like this:
```
  --all                  do output all messages including uiDebug, userFlag.
                         this may causes issues when used along with UI
```